src/ui/components/InputRequest/InputRequest.tsx

```diff
@@ -59,7 +59,7 @@ const InputRequest = () => {
     // This will be removed when the create connection process ends.
     const connectionName = new URL(content).searchParams.get("name");
 
-    const pendingId = uuidv4();
+    const pendingId = new Salter({}).qb64;
```

Comment:
Can we write a function in services/utils.ts to get a `randomSalt` and expose that to reuse everywhere? That way the UI doesn't need to import signify-ts, in case we want to decouple them.